app/pages/project/instances/actions.tsx

```diff
@@ -65,14 +67,28 @@ export const useMakeInstanceActions = (
         {
           label: 'Stop',
           onActivate() {
-            stopInstance.mutate(instanceParams, {
-              onSuccess: () => addToast({ title: `Stopping instance '${instance.name}'` }),
-              onError: (error) =>
-                addToast({
-                  variant: 'error',
-                  title: `Error stopping instance '${instance.name}'`,
-                  content: error.message,
+            confirmAction({
+              actionType: 'danger',
+              doAction: async () =>
+                stopInstance.mutate(instanceParams, {
+                  onSuccess: () =>
+                    addToast({ title: `Stopping instance '${instance.name}'` }),
+                  onError: (error) =>
+                    addToast({
+                      variant: 'error',
+                      title: `Error stopping instance '${instance.name}'`,
+                      content: error.message,
+                    }),
                 }),
+              modalTitle: 'Confirm stop',
```

Comment:
This matches the others, but I wonder if maybe these should say the word "instance"? Like "Confirm stop instance", "Confirm delete instance", etc. I like being redundant about what the thing under consideration is.